client/src/components/RatingsComponents/ReviewsList/NewReviewForm.jsx

```diff
@@ -53,7 +53,10 @@ function NewReviewForm ({factors, productName, closeModalOnSubmit, characteristi
     Axios({
       method: 'post',
       url: `http://127.0.0.1:3000/reviews?product_id=${product_Id}`,
```

Comment:
I'd imagine that the addition of the params below would mean that the url on Line 55 should stop at 'reviews'.

Reply:
yes! good catch!